packages/openapi-to-graphql/src/auth_builder.ts

```diff
@@ -244,7 +247,8 @@ function getViewerOT<TSource, TContext, TArgs>(
 function getViewerAnyAuthOT<TSource, TContext, TArgs>(
   name: string,
   queryFields: GraphQLFieldConfigMap<any, any>,
-  data: PreprocessingData<TSource, TContext, TArgs>
+  data: PreprocessingData<TSource, TContext, TArgs>,
+  fetch: typeof global['fetch']
```

Comment:
Why is this `typeof global['fetch']` instead of `typeof crossFetch`?

Reply:
Fixed :)